assignments/prototypes.js

```diff
@@ -92,19 +98,23 @@
     language: 'Elvish',
   });
 
-  console.log(mage.createdAt); // Today's date
-  console.log(archer.dimensions); // { length: 1, width: 2, height: 4 }
-  console.log(swordsman.healthPoints); // 15
-  console.log(mage.name); // Bruce
-  console.log(swordsman.team); // The Round Table
-  console.log(mage.weapons); // Staff of Shamalama
-  console.log(archer.language); // Elvish
-  console.log(archer.greet()); // Lilith offers a greeting in Elvish.
-  console.log(mage.takeDamage()); // Bruce took damage.
-  console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
-*/
+   console.log(mage.createdAt); // Today's date
+   console.log(archer.dimensions); // { length: 1, width: 2, height: 4 }
```

Comment:
This one should be printing an object with the keys being length, width, and height for the dimensions of your character. Currently this will print out only a single array with hard coded strings. This might not be what you want as you won't be able to alter or use these values at all in your game.